js/kucoin.js

```diff
@@ -3,7 +3,7 @@
 // ---------------------------------------------------------------------------
 
 const Exchange = require ('./base/Exchange');
-const { ExchangeError, ArgumentsRequired, ExchangeNotAvailable, InsufficientFunds, OrderNotFound, InvalidOrder, AccountSuspended, InvalidNonce, NotSupported, BadRequest, AuthenticationError, BadSymbol, RateLimitExceeded, PermissionDenied } = require ('./base/errors');
```

Comment:
is it possible to move all the edits related to kucoinfutures into the new exchange class kucoinfutures? would make this much easier to merge

Reply:
you can keep the removal of `fetchPositions` and `fetchFundingHistory` since those should be in kucoinfutures anyway

Reply:
I moved most of them over, just say the word and I'll move the other ones, but I thought it was better if I didn't